ourhexenv.py

```diff
@@ -57,14 +57,25 @@ def __init__(self, board_size=5, sparse_flag=True):
 
     def reset(self):
         self.board = np.zeros((self.board_size, self.board_size), dtype=int)
+        
+        self.isFirst = True
```

Comment:
Python uses snakecase instead of camelCase. Consider changing this to is_first instead

Reply:
Got it thanks